socorro/external/postgresql/crashstorage.py

```diff
@@ -292,32 +279,6 @@ def _get_raw_crash_transaction(self, connection, crash_id):
         except SQLDidNotReturnSingleValue:
             raise CrashIDNotFound(crash_id)
 
-    def get_unredacted_processed(self, crash_id):
-        """this method returns an unredacted processed crash
-
-        parameters:
-           crash_id - the id of a crash to fetch"""
-        return self.transaction(
-            self._get_processed_crash_transaction,
-            crash_id
-        )
-
-    def _get_processed_crash_transaction(self, connection, crash_id):
-        processed_crash_table_name = (
-            'processed_crashes_%s' % self._table_suffix_for_crash_id(crash_id)
-        )
-        fetch_sql = 'select processed_crash from %s where uuid = %%s' % \
-                    processed_crash_table_name
-        try:
-            return single_value_sql(connection, fetch_sql, (crash_id,))
-        except ProgrammingError as e:
-            err = 'relation "%s" does not exist' % processed_crash_table_name
-            if err in str(e):
-                raise CrashIDNotFound(crash_id)
-            raise
-        except SQLDidNotReturnSingleValue:
-            raise CrashIDNotFound(crash_id)
```

Comment:
Since we're no longer storing in the `processed_crashes_*` tables, we needed to ditch these two methods which fetch from them, too.

These were only used by the `CrashData` class which goes away in this PR, too.